packages/nuxt/package.json

```diff
@@ -40,6 +40,7 @@
     "@nuxt/schema": "latest",
     "@nuxt/vite-builder": "latest",
     "@nuxtjs/eslint-config-typescript": "latest",
-    "eslint": "latest"
+    "eslint": "latest",
+    "when-dependencies-installed": "^1.0.0"
```

Comment:
Should we add the same script in this package as in the sandbox package?
`"dependenciesInstalled": "node ./scripts/stud-nuxt.mjs"`